imxrt1060-hal/src/gpio.rs

```diff
@@ -71,6 +71,13 @@ where
         1u32 << <P as Pin>::Offset::USIZE
     }
 
+    /// Returns the ICR field offset for this GPIO
+    ///
+    /// ICR is "Interrupt Clear Register"
```

Comment:
ICR is the interrupt _configuration_ register, accepting `InterruptConfiguration` enum values. We'll clear interrupts with the status register, ISR.
```suggestion
    /// ICR is "Interrupt Configuration Register"
```